docs/contributing.md

```diff
@@ -0,0 +1,30 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Contributing to Submariner](#contributing-to-submariner)
+  - [Overview](#overview)
+  - [How to contribute](#how-to-contribute)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Contributing to Submariner
+
+:+1::tada: First off, thanks for taking the time to contribute to Submariner! :tada::+1:
+
+## Overview
+
+Submariner is a tool built to connect overlay networks of different Kubernetes clusters. It is CNI agnostic.
+
+## How to contribute
+
```

Comment:
Sign up to our mailing list here: https://groups.google.com/forum/#!forum/submariner-dev  and say hi :-D

Reply:
Join https://rancher-users.slack.com/messages/CGY62QBAR #submariner channel

Reply:
Ya forgot about slack channel. I am not sure if Google groups are active.

Reply:
The Google groups were created a couple of days ago ;-). The main driver there was that operators on OperatorHub are required to have a support mailing list.

Reply:
Correct, we need to tell people somewhere about their existance :-)